packages/loopback/lib/context/index.ts

```diff
@@ -2,7 +2,10 @@
 // Node module: loopback
 // This file is licensed under the MIT License.
 // License text available at https://opensource.org/licenses/MIT
-import {Binding} from './binding';
+import {Binding, BoundValue} from './binding';
+import {inject, describeInjectedArguments} from './inject';
+
+export {inject};
 
 export class Context {
   private registry: Map<string, Binding>;
```

Comment:
> Should we support bindings using `class` or `interface` as keys?

Let's leave that out of scope of this pull request to keep it small and focused. 

I find the approach used in https://github.com/joelday/decoration-ioc (Visual Studio Code) as an interesting way for avoiding stringly-typed binding keys.